07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,62 @@
+body {
+  width: 850px;
+}
+#sec1 {
+  padding-top: 225px;
+  text-align:center;
+  height: 225px;
+  background-image: url(../img/nasa-background.jpg);
+}
```

Comment:
don't forget to account for how high your text is. the H1 is still off-center because it has a height.